### PR TITLE
Fixed #27842 -- Allowed specifying 'protocol' in GenericSitemap's constructor.

### PR DESCRIPTION
**Pull request by chexex**
[ticket #27842](https://code.djangoproject.com/ticket/27842)

### CODE REVIEW

File: docs/ref/contrib/sitemaps.txt [hunk 1]
```diff
@@ -264,14 +264,20 @@ The sitemap framework provides a convenience class for a common case:
...
+    of the sitemap. It may also have ``date_field`` and
+    :attr:`~Sitemap.protocol` entries. ``Date_field`` specifies a date
+    field for objects retrieved from the ``queryset``.
     This will be used for the :attr:`~Sitemap.lastmod` attribute in the
     generated sitemap. You may also pass :attr:`~Sitemap.priority` and
     :attr:`~Sitemap.changefreq` keyword arguments to the
     :class:`~django.contrib.sitemaps.GenericSitemap`  constructor to specify
     these attributes for all URLs.
 
+.. versionadded:: 1.11
```
Comment (auvipy) [inline docs/ref/contrib/sitemaps.txt]: should this be 2.0?

File: docs/releases/1.11.txt [hunk 2]
```diff
@@ -223,6 +223,12 @@ Minor features
   :class:`~django.contrib.postgres.forms.HStoreField` allow storing null
   values.
 
+:mod:`django.contrib.sitemaps`
```
Comment (auvipy) [inline docs/releases/1.11.txt]: I think this should be moved to 2.0.txt as 1.11 is feature freeze

**Comment by timgraham** [comment]
It's not clear to me that 3 tests are needed for a 1 line change. I think we can assume that if `self.protocol` is set, it works correctly given this should be tested by the superclass.

**Comment by timgraham** [comment]
Did you use some criteria in deciding to put this in `info_dict` rather than as a separate keyword arguments like `priority` and `changefreq`?

**Comment by chexex** [comment]
That's a good question. 
Since last week I think that info_dict used to specify special non common for Sitemap class arguments. So, may be, there will be better to pass "protocol" as a kwarg along with "priority" and "changefreq". Isn't it?

**Comment by timgraham** [comment]
From a quick look, that was also the conclusion I came to.

**Pull request merged**